pulumi/folio/Pulumi.dev.yaml

```diff
@@ -16,4 +16,4 @@ config:
   folio:superuser-name:
     secure: v1:HrzjbPVTBhV6QvOy:ZMIqcIZAVAO0hd3Q1tBv1kBtu/h4L4ICLJ4=
   folio:superuser-password:
-    secure: v1:qx/xoBErsTMhCxYk:kaLCmhpXa8sJdD4z4cWfftKyXRr8bvDKb+b+4gLvMRyXSQdQR2lEDjtH98gAQvc=
+    secure: v1:6RsFh9tFbr5Ymo4X:OhxdfTFo0MUWhk5NeFZspbCcFbBidVUlNN/m7BawrFGcorgVFAlOTn5S98AQge0=
```

Comment:
Not sure what happened here. Maybe I changed this pw in the dev stack (our current production stack) by mistake before switching to the new stack. Let me see if I can switch it back.

Reply:
Well I checked the actual value by running `pulumi config --show-secrets` for the stack and it's the same one that is in keepass so not sure what would cause the hash to change.